package-lock.json

```diff
@@ -111,11 +112,7 @@
         "babel-external-helpers": "bin/babel-external-helpers.js"
       },
       "optionalDependencies": {
-        "@nicolo-ribaudo/chokidar-2": "2.1.8-no-fsevents",
-        "chokidar": "^3.4.0"
```

Comment:
This dependency can cause npm install failures when babel is installed, so when installing non-production modules. That's why we need to be so careful with the lock version being 2 and to install only prod modules.